### ISSUE TITLE
Importing large csv using 'get data' results in failure

### ISSUE THREAD

**Issue by thibaultmol** [ONLYOFFICE/DocumentServer#1898]
### This issue is unique.

- [X] I have used the [search tool](https://github.com/ONLYOFFICE/DesktopEditors/issues?q=) and did not find an issue describing my bug.

### Operating System

Linux (DEB package)

### Version information

7.1.1.57

### Expected Behavior

Import the csv file correctly

### Actual Behavior

![image](https://user-images.githubusercontent.com/1010226/188928978-348efa13-5791-43bd-883f-70eab4467170.png)

### Reproduction Steps

- Open Onlyoffice
- Create new spreadsheet
- Download csv test file from here https://eforexcel.com/wp/downloads-18-sample-csv-files-data-sets-for-testing-sales/
Use the '100000 Sales Records' one
- Go to 'Data' 'Get data' and select the csv file you just downloaded.

### Additional information

Operating System: Garuda Linux
KDE Plasma Version: 5.25.5
KDE Frameworks Version: 5.97.0
Qt Version: 5.15.5
Kernel Version: 5.19.3-zen1-1-zen (64-bit)
Graphics Platform: X11
Processors: 16 × AMD Ryzen 7 3700X 8-Core Processor
Memory: 31,3 GiB of RAM
Graphics Processor: NVIDIA GeForce GTX 1080/PCIe/SSE2

Installed via the AUR but also uninstalled that version and installed flatpak version instead. both have this issue
(btw, flatpak isn't an option un 'version information' in this bug report form

**Comment by ShockwaveNN** [comment]
Moving this issue to DocumentServer repo, since it's our base product and problem actual there too

**Comment by thibaultmol** [comment]
> I can confirm - this is bug, issue 58948 in our private issue tracker.
> Thank you for report

from @VladimirProkofiev52  on a different issue

**Comment by ShockwaveNN** [comment]
Yep, this comment is also moved. Seems there were some problems in github side for moving this issue. Usually it's several seconds, but was around 15 minutes this time
